app/assets/fonts/bariol_light_italic-demo.html

```diff
@@ -0,0 +1,613 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
@rychkog  all copied from old page ( like all styles and part of templates ) ( all `fonts` directory ). if we still be using these fonts I can clean it up